cfme/common/vm.py

```diff
@@ -659,17 +658,19 @@ def wait_candu_data_available(self, timeout=600):
             delay=10, handle_exception=True, num_sec=timeout,
             fail_func=view.toolbar.reload.click)
 
-    def capture_historical_data(self, interval='hourly', back='6.days'):
+    def capture_historical_data(self, interval="hourly", back="6.days"):
         """Capture historical utilization data for this VM/Instance
 
         Args:
             interval: Data interval (hourly/ daily)
             back: back time interval from which you want data
         """
         ret = self.appliance.ssh_client.run_rails_command(
-            "\"vm = Vm.where(:ems_id => {}).where(:name => {})[0];\
-            vm.perf_capture({}, {}.ago.utc, Time.now.utc)\""
-            .format(self.provider.id, repr(self.name), repr(interval), back))
+            "\"vm = Vm.where(:ems_id => {prov_id}).where(:name => '{vm_name}')[0];\
+            vm.perf_capture('{interval}', {back}.ago.utc, Time.now.utc)\"".format(
+                prov_id=self.provider.id, vm_name=self.name, interval=interval, back=back
```

Comment:
**optional** for both of name and interval i would like to propose to use a `json.dump` to protect from bad characters